.idea/prettier.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="PrettierConfiguration">
+    <option name="myRunOnSave" value="true" />
+    <option name="myRunOnReformat" value="true" />
+  </component>
+</project>
```

Comment:
remove?

Reply:
We have committed other files in the `.idea` folder so I figured this configuration was pretty harmless.